### PR TITLE
[networkx] Supplement graphscope.nx.{readwrite, convert, drawing} with more functions and test cases

### LINKED ISSUE #979: [NetworkX] Supplement functions and test cases of networkx  module
current networkx module only contains classes and algorithms test cases, we need to supplement functions and test cases of other submodule and make them pass.
- [x] generator
- [x] readwrite
- [ ] drawing
- [ ] linalg (low priority)

### PR DESCRIPTION
**Pull request by acezen**
## What do these changes do?
Supplement graphscope.nx.{readwrite, convert} with more functions and test cases

## Related issue number

fixed #979

### CODE REVIEW

File: python/graphscope/nx/readwrite/multiline_adjlist.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+#
+# NetworkX is distributed under a BSD license; see LICENSE.txt for more
+# information.
+#
+
+import networkx.readwrite.multiline_adjlist
+
+from graphscope.nx.utils.compat import import_as_graphscope_nx
+
+import_as_graphscope_nx(networkx.readwrite.multiline_adjlist)
```
Comment (sighingnow) [inline python/graphscope/nx/readwrite/multiline_adjlist.py:26]: We use `import_as_graphscope_nx` here that this file is no longer "derived" from networkx.

**Review by sighingnow — CHANGES_REQUESTED** [review]
Please re-check the license header in the newly added `test_xxxx.py` files.

**Comment by acezen** [comment]
> Please re-check the license header in the newly added `test_xxxx.py` files.

thanks for pointing out, already update the license

File: python/graphscope/nx/readwrite/multiline_adjlist.py (hunk 1 above)
Comment (acezen) [reply python/graphscope/nx/readwrite/multiline_adjlist.py:23]: fixed

**Review by sighingnow — APPROVED** [review]
LGTM.

**Pull request merged**